### ISSUE TITLE
feat(datepicker): override weekday render via prop/slot

### ISSUE THREAD

**Issue by deiiviidas** [tusen-ai/naive-ui#5695]
### Describe the bug
removed becouse everything here is feature and not a bug

**Comment by Volankey** [comment]
See https://www.naiveui.com/en-US/os-theme/docs/i18n

**Comment by deiiviidas** [comment]
> See https://www.naiveui.com/en-US/os-theme/docs/i18n

Then please, give me an example, how to change/translate weekdays to my custom? for example mondays/sundays?

**Comment by deiiviidas** [comment]
i did this dirty fix. but it should be easy customizable becouse it sucks if i cant change my weekdays to any of the name.
language it is not supported.


![JEmtxg](https://github.com/tusen-ai/naive-ui/assets/25092057/788a0bea-5c34-4d83-87de-c1eee839db42)


![WzgM8U](https://github.com/tusen-ai/naive-ui/assets/25092057/1a037723-45d6-4794-a688-2d1b649d4e93)

**Comment by Volankey** [comment]
Use 
```
import { dateItIT, itIT } from "naive-ui";
```

**Comment by deiiviidas** [comment]
i need ltLT

2024-02-21, tr, 04:47 Volankey ***@***.***> rašė:

**Comment by deiiviidas** [comment]
basically it is limited to locale and you cant change. - cant change, unless it is from locale

**Comment by Volankey** [comment]
Yes, it can't be changed any other way, we don't have the resources to do it.
PRs are welcomed for locales that are not supported yet.

> basically it is limited to locale and you cant change. - cant change, unless it is from locale

**Comment by deiiviidas** [comment]
![m5vpBx](https://github.com/tusen-ai/naive-ui/assets/25092057/ee7f3e37-272e-4181-a757-067d474cbf02)

**Comment by deiiviidas** [comment]
i gave atleast 5 different examples here how to do it.

**Comment by deiiviidas** [comment]
```
<n-date-picker
    placeholder="dd/mm/yyyy"
    format="dd/MM/yyyy"
    :weekdays="['P', 'A', 'T', 'K', 'Pn', 'Š', 'S']"
/>
```

another OP example

**Comment by jahnli** [comment]
No plans yet, but PRs are welcome
